### ISSUE TITLE
Unable to Operate Cloudstack property in EL9 with UEFI Secure boot and libvirt 9.5

### ISSUE THREAD

**Issue by hmadra** [apache/cloudstack#8280]
* Bug Report
 
##### COMPONENT NAME
General Operations

##### CLOUDSTACK VERSION

~~~
4.18.1
~~~

##### OS / ENVIRONMENT

AlmaLinux 9.3 up-to-date 
libvirt 9.5

##### SUMMARY

Libvirt has made iTCO as a default watchdog for q35 based VMs,  added to all the XML definitions on KVM to each VM.
Cloudstack adds its own watchdog i6300esb etc for this model.

With 2 watchdogs in the vm definition , cloudstack agent is not able to parse the vm xml when a request is made, post creation.
This means , no operation including Migration , Console , Shutdown etc operate 

Here is the exception from CS Agent on the host
~~~
2023-11-29 18:33:03,740 WARN  [cloud.agent.Agent] (agentRequest-Handler-4:null) (logid:2b400d1e) Caught:
java.lang.IllegalArgumentException: No enum constant com.cloud.hypervisor.kvm.resource.LibvirtVMDef.WatchDogDef.WatchDogModel.ITCO
        at java.base/java.lang.Enum.valueOf(Enum.java:240)
        at com.cloud.hypervisor.kvm.resource.LibvirtVMDef$WatchDogDef$WatchDogModel.valueOf(LibvirtVMDef.java:2159)
        at com.cloud.hypervisor.kvm.resource.LibvirtDomainXMLParser.parseDomainXML(LibvirtDomainXMLParser.java:332)
        at com.cloud.hypervisor.kvm.resource.LibvirtComputingResource.getInterfaces(LibvirtComputingResource.java:4012)
        at com.cloud.hypervisor.kvm.resource.LibvirtComputingResource.getVmStat(LibvirtComputingResource.java:4206)
        at com.cloud.hypervisor.kvm.resource.wrapper.LibvirtGetVmStatsCommandWrapper.execute(LibvirtGetVmStatsCommandWrapper.java:53)
        at com.cloud.hypervisor.kvm.resource.wrapper.LibvirtGetVmStatsCommandWrapper.execute(LibvirtGetVmStatsCommandWrapper.java:37)
        at com.cloud.hypervisor.kvm.resource.wrapper.LibvirtRequestWrapper.execute(LibvirtRequestWrapper.java:78)
        at com.cloud.hypervisor.kvm.resource.LibvirtComputingResource.executeRequest(LibvirtComputingResource.java:1848)
        at com.cloud.agent.Agent.processRequest(Agent.java:662)
        at com.cloud.agent.Agent$AgentRequestHandler.doTask(Agent.java:1082)
        at com.cloud.utils.nio.Task.call(Task.java:83)
        at com.cloud.utils.nio.Task.call(Task.java:29)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
~~~

##### STEPS TO REPRODUCE

Update EL9 to current package version.
Run a UEFI based VM.

##### EXPECTED RESULTS
All operations should run normally.

##### ACTUAL RESULTS
VMs cannot be operated as required.

**Comment by DaanHoogland** [comment]
@hmadra can you test if #8282 works for you?

**Comment by weizhouapache** [comment]
fixed by #8282
